hadoop-ozone/dist/src/main/smoketest/topology/cli.robot

```diff
@@ -22,18 +22,27 @@ Test Timeout        5 minutes
 
 *** Variables ***
 
-
 *** Test Cases ***
 Run printTopology
     ${output} =         Execute          ozone admin printTopology
-                        Should contain   ${output}         10.5.0.7(ozone-topology_datanode_4_1.ozone-topology_net)    IN_SERVICE    /rack2
+                        Should Contain   ${output}         State = HEALTHY
+                        Should Contain          ${output}         IN_SERVICE
+                        Should Match Regexp     ${output}         .*datanode[-_]\\d+.*IN_SERVICE.*
```

Comment:
`State = HEALTHY` was not in the original expected content, why is it added?

Why is check for `rack2` removed?

Reply:
Each test case begins by checking for the State = HEALTHY output, ensuring that the overall state of the system or the datanodes is as expected.
All the commands start with this line. 

```
sh-4.2$ ozone admin printTopology
State = HEALTHY
 172.19.0.8(ozone-datanode-4.ozone_default):HTTP=9882,CLIENT_RPC=19864,REPLICATION=9886,RATIS=9858,RATIS_ADMIN=9857,RATIS_SERVER=9856,RATIS_DATASTREAM=9855,STANDALONE=9859    IN_SERVICE    /default-rack
 172.19.0.10(ozone-datanode-1.ozone_default):HTTP=9882,CLIENT_RPC=19864,REPLICATION=9886,RATIS=9858,RATIS_ADMIN=9857,RATIS_SERVER=9856,RATIS_DATASTREAM=9855,STANDALONE=9859    IN_SERVICE    /default-rack
 172.19.0.7(ozone-datanode-5.ozone_default):HTTP=9882,CLIENT_RPC=19864,REPLICATION=9886,RATIS=9858,RATIS_ADMIN=9857,RATIS_SERVER=9856,RATIS_DATASTREAM=9855,STANDALONE=9859    IN_SERVICE    /default-rack
```
And I have added the check for rack again